contacts/index.js

```diff
@@ -77,10 +77,11 @@ route.delete("/:id", authenticate, (req, res) => {
                 .del()
                 .then(result => {
                   if (result) {
-                    res.json({
-                      message: "Contact deleted successfully",
-                      success: true
-                    });
+                    db("contacts")
+                      .whereNot({ group_id: id })
```

Comment:
am I wrong, or are you sending every contact in the whole database except for the deleted one here? I don't get why you would send that? If anything, you would want to send contacts limited to the specific user or that organization. But I don't think you need to send anything except a success status to be honest.